SetupDataPkg/Library/ConfigDataLib/UnitTest/ConfigDataLibUnitTest.c

```diff
@@ -0,0 +1,327 @@
+/** @file
+  Unit tests of the ConfigDataLib instance.
+
+  Copyright (C) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <stdio.h>
+#include <string.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+#include <Uefi.h>
+#include <Library/BaseLib.h>
+#include <Library/DebugLib.h>
+#include <Library/ConfigDataLib.h>
+
+#include <Library/UnitTestLib.h>
+
+#include <Good_Config_Data.h>
+
+#define UNIT_TEST_APP_NAME     "Conf Data Iterator Unit Tests"
+#define UNIT_TEST_APP_VERSION  "1.0"
+
+/**
+  Handler function dispatched for individual tag based data.
+
+  @param[in] ConfDataPtr  Pointer to configuration data.

```

Comment:
The parameter here isn't any of the ones listed in the actual function.  Also none of the actual input parameters are described.

Reply:
Thanks, updated.